src/test/java/uk/gov/pay/ledger/transaction/resource/TransactionResourceSearchIT.java

```diff
@@ -408,4 +408,73 @@ public void shouldSearchCorrectlyOnTransactionAndParentTransaction_WhenWithParen
                 .body("results[1].transaction_id", is(payment.getExternalId()))
                 .body("results[1].parent_transaction", is(nullValue()));
     }
+
+    @Test
+    public void shouldSearchUsingMultipleGatewayAccountIds() {
+        String targetGatewayAccountId = "123";
+        String otherGatewayAccountId = "456";
+
+        TransactionFixture targetPayment = aTransactionFixture()
+                .withTransactionType("PAYMENT")
+                .withState(TransactionState.SUBMITTED)
+                .withGatewayAccountId(targetGatewayAccountId)
+                .insert(rule.getJdbi());
+
+        aTransactionFixture()
+                .withTransactionType("PAYMENT")
+                .withState(TransactionState.SUBMITTED)
+                .withGatewayAccountId(otherGatewayAccountId)
+                .insert(rule.getJdbi());
+
+        given().port(port)
+                .contentType(JSON)
+                .accept(JSON)
+                .get("/v1/transaction?" +
+                        "account_id=" + targetGatewayAccountId+ "," + otherGatewayAccountId+
+                        "&page=1" +
+                        "&display_size=5"
+                )
+                .then()
+                .statusCode(Response.Status.OK.getStatusCode())
+                .contentType(JSON)
+                .body("count", is(2));
+    }
+
+    @Test
+    public void shouldSearchUsingMultipleGatewayAccountIdsExclusively() {
+        String targetGatewayAccountId = "123";
+        String otherGatewayAccountId = "456";
+        String otherestGatewayAccountId = "1337";
+
+        TransactionFixture targetPayment = aTransactionFixture()
+                .withTransactionType("PAYMENT")
+                .withState(TransactionState.SUBMITTED)
+                .withGatewayAccountId(targetGatewayAccountId)
+                .insert(rule.getJdbi());
+
+        aTransactionFixture()
+                .withTransactionType("PAYMENT")
+                .withState(TransactionState.SUBMITTED)
+                .withGatewayAccountId(otherGatewayAccountId)
+                .insert(rule.getJdbi());
+
+        aTransactionFixture()
+                .withTransactionType("PAYMENT")
+                .withState(TransactionState.SUBMITTED)
+                .withGatewayAccountId(otherestGatewayAccountId)
+                .insert(rule.getJdbi());
+
+        given().port(port)
+                .contentType(JSON)
+                .accept(JSON)
+                .get("/v1/transaction?" +
+                        "account_id=" + targetGatewayAccountId+ "," + otherGatewayAccountId+
```

Comment:
Looks like the `otherGatewayAccountId` is also a `targetGatewayAccountId`, really.